lib/src/main/java/com/orange/ods/compose/component/button/OdsIconButton.kt

```diff
@@ -37,129 +32,29 @@ import com.orange.ods.compose.theme.OdsTheme
- */
-@Composable
-@OdsComposable
-fun OdsIconButton(
-    onClick: () -> Unit,
-    imageVector: ImageVector,
-    contentDescription: String,
-    modifier: Modifier = Modifier,
-    enabled: Boolean = true,
-    displaySurface: OdsDisplaySurface = OdsDisplaySurface.Default
-) {
-    OdsIconButton(
-        onClick = onClick,
-        graphicsObject = imageVector as Any,
-        contentDescription = contentDescription,
-        modifier = modifier,
-        enabled = enabled,
-        displaySurface = displaySurface
-    )
-}
-
-/**
- * OdsIconButton is a clickable icon, used to represent actions. An OdsIconButton has an overall minimum
- * touch target size of 48 x 48dp, to meet accessibility guidelines. It contains an [Icon] centered
- * inside the OdsIconButton.
- * If using a custom icon, note that the typical size for the internal icon is 24 x 24 dp.
- *
- * This component is typically used inside an App Bar for the navigation icon / actions.
- *
- * @param onClick the lambda to be invoked when this icon is pressed
- * @param bitmap [ImageBitmap] to draw inside this OdsIconButton
- * @param contentDescription the content description associated to this OdsIconButton.
- * @param modifier optional [Modifier] for this OdsIconButton
- * @param enabled whether or not this OdsIconButton will handle input events and appear enabled for
- * semantics purposes, true by default
- * @param displaySurface optional allow to force the button display on a dark or light
- * surface. By default the appearance applied is based on the system night mode value.
- */
-@Composable
-@OdsComposable
-fun OdsIconButton(
-    onClick: () -> Unit,
-    bitmap: ImageBitmap,
-    contentDescription: String,
-    modifier: Modifier = Modifier,
-    enabled: Boolean = true,
-    displaySurface: OdsDisplaySurface = OdsDisplaySurface.Default
-) {
-    OdsIconButton(
-        onClick = onClick,
-        graphicsObject = bitmap as Any,
-        contentDescription = contentDescription,
-        modifier = modifier,
-        enabled = enabled,
-        displaySurface = displaySurface
-    )
-}
-
-@Composable
-internal fun OdsIconButton(
-    onClick: () -> Unit,
-    graphicsObject: Any,
-    contentDescription: String,
     modifier: Modifier = Modifier,
     enabled: Boolean = true,
     displaySurface: OdsDisplaySurface = OdsDisplaySurface.Default
 ) {
     CompositionLocalProvider(
         LocalRippleTheme provides displaySurface.rippleTheme
     ) {
-        OdsIconButton(
-            onClick = onClick,
-            graphicsObject = graphicsObject,
-            contentDescription = contentDescription,
-            modifier = modifier.background(color = iconButtonBackgroundColor(displaySurface = displaySurface)),
-            enabled = enabled,
-            tint = iconButtonTintColor(displaySurface = displaySurface)
-        )
+        IconButton(onClick = onClick, modifier = modifier, enabled = enabled) {
```

Comment:
I wonder if we could factorize the two `OdsIconButton` methods but it does not seem easy at all.

Reply:
The first one is for the user with a public API  and the second one is internal with graphicsObject...